fe/src/component/Tab/Tab.js

```diff
@@ -1,43 +1,79 @@
 import List from "./List";
-import "./Tab.css";
-import React, { useState } from "react";
+import Card from "../UI/Card";
+import CardsWrapper from "../UI/CardsWrapper";
+import styled from "styled-components";
+import React, { useEffect, useState } from "react";
+
+const TabList = styled.ul`
+  display: flex;
+  margin: 20px 0 20px 40px;
+`;
+
+const TabListItem = styled.li`
+  margin-right: 30px;
+  font-size: 20px;
+  font-weight: 500;
+  cursor: pointer;
+  &.active {
+    border-bottom: 1px solid #000;
+  }
+`;
 
 const Tab = () => {
   const [infor, setInfor] = useState([
-    { id: 1, title: "풍성한 고기반찬", active: true },
-    { id: 2, title: "편리한 반찬세트", active: false },
-    { id: 3, title: "맛있는 제철요리", active: false },
-    { id: 4, title: "우리 아이 영향 반찬", active: false },
+    { id: 1, title: "풍성한 고기반찬" },
+    { id: 2, title: "편리한 반찬세트" },
+    { id: 3, title: "맛있는 제철요리" },
+    { id: 4, title: "우리 아이 영향 반찬" },
   ]);
 
-  const onChangeInfor = (clickedId) => {
-    const newInfor = infor.map((v) => {
-      return Object.assign(v, { active: false });
-    });
-    const newnew = newInfor.map((v) =>
-      v.id.toString() === clickedId
-        ? (v = Object.assign(v, { active: true }))
-        : v
-    );
-    setInfor(newnew);
+  const [cards, setCards] = useState([]);
+  const [activeTab, setActiveTab] = useState(1);
+
+  useEffect(() => {
+    setActiveTab("1");
+  }, []);
+
+  useEffect(() => {
+    fetch("https://api.codesquad.kr/onban/main")
+      .then((res) => res.json())
+      .then((data) => setCards(data.body[activeTab]));
+  }, [activeTab]);
+
+  const onClickHandler = (event) => {
+    setActiveTab(event.target.id);
   };
 
   return (
-    <div>
-      <ul className="tab-list">
+    <>
+      <TabList>
         {infor.map((v) => {
           return (
-            <List
-              title={v.title}
-              key={v.id}
+            <TabListItem
               id={v.id}
-              className={v.active ? "active" : ""}
-              onSaveClickedID={onChangeInfor}
-            />
+              key={v.id}
+              className={v.id.toString() === activeTab ? "active" : ""}
+              onClick={onClickHandler}
+            >
+              {v.title}
+            </TabListItem>
           );
         })}
-      </ul>
-    </div>
+      </TabList>
+
+      <CardsWrapper>
+        <Card
+          key={cards.detail_hash}
+          image={cards.image}
+          alt={cards.alt}
+          title={cards.title}
+          description={cards.description}
+          s_price={cards.s_price}
+          n_price={cards.n_price}
+          badge={cards.badge}
```

Comment:
```suggestion
          key={cards.detail_hash}
          {...cards}
```
react는 props를 스프레드시켜서 표현 가능합니다 !  
https://kevinyckim33.medium.com/jsx-spread-operator-component-props-meaning-3c9bcadd2493